### DISCUSSION TITLE
Is ezdxf suitable for trim/crop dxf file?

### DISCUSSION THREAD

**Discussion by longgangfan** [mozman/ezdxf#843] [Q&A]
As the Title above, is there any way to crop a dxf file with ezdxf? can you give an example?

**Comment by mozman** [comment]
No,. this is not a design goal of this package.

**Reply by longgangfan** [reply]
Thank you, I find GDAL command ogr2ogr can do this job.

**Reply by asd513921** [reply]
Please ask how to solve this problem, I also need to implement this function
